lib/runcible/resources/consumer.rb

```diff
@@ -7,7 +7,7 @@ class Consumer < Runcible::Base
       # @param  [String]  id  the ID of the consumer
       # @return [String]      the consumer path, may contain the id if passed
       def self.path(id = nil)
-        (id.nil?) ? 'consumers/' : "consumers/#{id}/"
+        id.nil? ? 'consumers/' : "consumers/#{id}/"
```

Comment:
Yea, I agree. Would you be in favor of disabling the `RedundantParentheses` cop, adding `rubocop:disable`s, or just leaving it as-is?